### DISCUSSION TITLE
How to disable the info level logs from terminal

### DISCUSSION THREAD

**Discussion by IshmeetSingh06** [rspec/rspec-rails#2750] [Q&A]
Whenever we run api/controller specs we keep getting these logs 
`level=INFO method=GET path=/api/v2/xyz status=401 duration=17.98 view=7.74 db=10.24 current_user_type= current_user_id= params={} datetime=2024-03-28T12:37:12.495570Z host=www.example.com authorization= user_type=`

these fill up our whole terminal is there any way to silence these logs, i've tried setting the config.log_level to :warn inside the config/environments/test.rb but it didnt work

**Comment by chayuto** [comment]
having the same issue
